lang/src/idl.rs

```diff
@@ -22,6 +22,9 @@ use crate::accounts::program_account::ProgramAccount;
 use crate::prelude::*;
 use solana_program::pubkey::Pubkey;
 
+// re-export CtorBumps
+pub use crate::ctor::CtorBumps;
```

Comment:
had to do this because `IdlCreateAccounts` is a type alias for `Ctor` so the generated bumps struct was `CtorBumps` instead of `IdlCreateAccountsBumps`. See https://github.com/coral-xyz/anchor/pull/2065/files#diff-40361330a9a3230c38138958631e0fa34181581c2cedcb74a4f6210067045851R28